default/generated/eapxp6/180-eapxp_microprofile_openapi_4.0.mta.yaml

```diff
@@ -0,0 +1,92 @@
+- ruleID: eapxp_microprofile_openapi_4.0-00001
+  category: mandatory
+  effort: 1
+  labels:
+  - konveyor.io/source=eapxp
+  - konveyor.io/target=eapxp6+
+  - konveyor.io/target=eapxp
+  message: Incompatible changes to the Schema model API, reflecting changes in the OpenAPI v3.1 document format.
```

Comment:
- There is a missing `description` field in each rule. The `description` field should contain something akin to a title for the issue.
- The `message` field should contain a more thorough message describing the problem, maybe including some explanation about how to fix it.

Reply:
Will update.
Please consider updating the rule documentation, which makes no mention of a description field for a rule: https://github.com/konveyor/analyzer-lsp/blob/main/docs/rules.md

Reply:
@pferraro thanks for highlighting the issue, addressed [here](https://github.com/konveyor/analyzer-lsp/pull/764).